aten/src/ATen/TensorIterator.cpp

```diff
@@ -831,6 +831,15 @@ void TensorIteratorBase::build_comparison_op(const Tensor& out, const Tensor& a,
   build(config);
 }
 
+void TensorIteratorBase::build_ternary_op(const Tensor& out, const Tensor& a,
+    const Tensor& b, const Tensor& c) {
+  build(TensorIteratorConfig()
```

Comment:
I agree with you. Those config make sense, at least for how `addcmul` and `addcdiv` should work. Not completely sure whether it would safe to do so, though.